### DISCUSSION TITLE
[Chapter 09] MONO? FLUX?

### DISCUSSION THREAD

**Discussion by ahngilwoong** [cowaine/spring-boot-for-msa#132] [Chapter 09. Application Integration: REST API] [answered]
책의 WebClient 설명 부분에서 Mono와 Flux의 개념이 나옵니다.
다루지 않는다고 말하긴 했지만 대충 Mono와 Flux의 차이가 뭔지도 안알려주고 모노로 바꿨다가 Flux로 바꿨다가 해서 처음 보는 입장에서는 사실 잘 이해하기 어려웠습니다.

1. 무조건 모노로 바꿨다가 Flux로 바꿔줘야 결과값을 받아 올 수 있는건가요?
2. 만약 1번이 아니라면 모노와 Flux의 차이가 무엇일까요?
3. 리액티브 타입의 예시가 있을까요 ?

**Comment by JoisFe** [comment, marked as answer]
저.. 해당 관련 공부중이긴한데.. 최대한 주석으로 적어두고 있어요

https://github.com/JoisFe/spring_webflux_start/tree/develop
